vignettes/read-myacc-csv.Rmd

```diff
@@ -0,0 +1,204 @@
+               rmc.dec = ".",
+               rmc.firstrow.acc = 2,
+               rmc.col.acc = 2:4,
+               rmc.col.temp = 5,
+               rmc.col.time=1,
+               rmc.unit.acc = "g",
+               rmc.unit.temp = "C",
+               rmc.unit.time = "POSIX",
+               rmc.format.time = "%d/%m/%Y %H:%M:%OS",
+               rmc.desiredtz = "Europe/London",
+               rmc.sf = 100)
+```
+
+## Example using the shell function
+
+If the `rmc.firstrow.acc` argument is defined within the `g.shell.GGIR` function, then the data will be read through `read.myacc.csv`. GGIR needs the user to specify in which row starts the accelerometer data within the csv, so this argument must be always explicitly specified by the user. Thus, a call to the `g.shell.GGIR` including the rmc arguments would look as follows (note that the `rmc.file` argument does not need to be specified as the datadir already contains such information).
+
+```{R,eval=FALSE}
+library(GGIR)
+g.shell.GGIR(
+             mode=c(1,2,3,4,5),
+             datadir="C:/mystudy/mydata/datafile.csv",
+             outputdir="D:/myresults",
+             do.report=c(2,4,5),
+             #=====================
+             # read.myacc.csv arguments
+             #=====================
+             rmc.nrow = Inf, 
+             rmc.dec = ".",
+             rmc.firstrow.acc = 2, 
+             rmc.col.acc = 2:4, 
+             rmc.col.temp = 5, 
+             rmc.col.time=1,
+             rmc.unit.acc = "g", 
+             rmc.unit.temp = "C", 
+             rmc.unit.time = "POSIX",
+             rmc.format.time = "%d/%m/%Y %H:%M:%OS",
+             rmc.desiredtz = "Europe/London",
+             rmc.sf = 100,
+             #=====================
+             # Part 1 
+             #=====================
+             rmc.noise = 0.013,
+             #=====================
+             # Part 2
+             #=====================
+             strategy = 1,
+             hrs.del.start = 0,          hrs.del.end = 0,
+             maxdur = 9,                 includedaycrit = 16,
+             qwindow=c(0,24),
+             mvpathreshold =c(100),
+             bout.metric = 6,
+             excludefirstlast = FALSE,
+             includenightcrit = 16,
+             #=====================
+             # Part 3 + 4
+             #=====================
+             def.noc.sleep = 1,
+             outliers.only = TRUE,
+             criterror = 4,
+             do.visual = TRUE,
+             #=====================
+             # Part 5
+             #=====================
+             threshold.lig = c(30), threshold.mod = c(100),  threshold.vig = c(400),
+             boutcriter = 0.8,      boutcriter.in = 0.9,     boutcriter.lig = 0.8,
+             boutcriter.mvpa = 0.8, boutdur.in = c(1,10,30), boutdur.lig = c(1,10),
+             boutdur.mvpa = c(1),
+             includedaycrit.part5 = 2/3,
+             #=====================
```

Comment:
Try to keep this short as possible, because for this vignette it does not matter how GGIR is used.
The shorter the g.shell.GGIR call the lower the risk that it has argument values that are not consistent with argument values elsewhere in the documentation. For example, remove all part 3, 4 and 5 arguments.